packages/paste-core/components/menu/__tests__/index.spec.tsx

```diff
@@ -80,6 +86,18 @@ describe('Menu', () => {
       expect(renderedMenuItem.getAttribute('role')).toEqual('menuitem');
     });
 
+    it('should render a menu item as a button', () => {
+      render(<MenuMock />);
+      const renderedMenuItem = screen.getByTestId('example-menu-item');
+      expect(renderedMenuItem.tagName).toBe('BUTTON');
+    });
+
+    it('should render a menu item as an anchor', () => {
+      render(<MenuMock />);
+      const renderedMenuItem = screen.getByTestId('example-menu-anchor');
+      expect(renderedMenuItem.tagName).toEqual('A');
+    });
+
```

Comment:
Worth adding tests that ensure the onClick handler is fired when I:

- click on an element
- press spacebar
- press enter

On both anchors and buttons?

Reply:
Yeah. Good call. Added those.